server/src/main/java/com/netcracker/domain/Message.java

```diff
@@ -0,0 +1,24 @@
+package com.netcracker.domain;
+
+public class Message {
+    private final MessageType messageType;
+    private final String data;
+
+    public Message(MessageType messageType) {
+        this.messageType = messageType;
+        data = "";
```

Comment:
&#x1F34E; **Issue**
You should be assigning class fields via this.variable = ... instead of variable = ... - it makes it more explicit when internal state of the class changes (and you want it to be as explicit, as possible, IDEA highlights class members, but its still not quite distinct enough).